astropy/modeling/functional_models.py

```diff
@@ -506,7 +506,7 @@ def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
                             ('amplitude', outputs_unit['z'])])
 
 
-class Shift(Model):
+class Shift(Fittable1DModel):
```

Comment:
Thanks, I hadn't spotted that. Although that PR was merged, there no longer seems to be any trace of it in the commit history, so presumably it was undone soon afterwards?